pkg/anonymization/anonymizer.go

```diff
@@ -362,17 +343,19 @@ func (anonymizer *Anonymizer) AnonymizeMemoryRecord(memoryRecord *record.MemoryR
 		}
 	})
 
-	if len(anonymizer.clusterBaseDomain) != 0 {
-		memoryRecord.Data = bytes.ReplaceAll(
-			memoryRecord.Data,
-			[]byte(anonymizer.clusterBaseDomain),
-			[]byte(ClusterBaseDomainPlaceholder),
-		)
-		memoryRecord.Name = strings.ReplaceAll(
-			memoryRecord.Name,
-			anonymizer.clusterBaseDomain,
-			ClusterBaseDomainPlaceholder,
-		)
+	if len(anonymizer.sensitiveValues) != 0 {
```

Comment:
I think this condition is not needed.

Reply:
Solved in bc66284efa66957e06efbc3efd2c1f893793085e, thanks for the review